src/pages/apply/project-manager/ApplyProjectManager.tsx

```diff
@@ -0,0 +1,179 @@
+        <meta
+          name="description"
+          content="Apply to be a Project Manager at Code4Community."
+        />
+      </Helmet>
+
+      <Hero
+        title="Project Manager"
+        subtitle="Create impactful software from the ground up. Lead with confidence."
+        SvgNode={SVG}
+      />
+
+      <Grid container spacing={3}>
+        <Grid item sm={7}>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="h5" gutterBottom>
+                What you'll be doing
+              </Typography>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                Project Managers at C4C are responsible for scoping out the features
+                of our projects, prioritizing developer work, and ensuring we're on track
+                to deliver a high quality product at the end of the year. Project Managers
+                lead team and client meetings and with the help of our tech leads adjust and
+                optimize the team's focus and process. They are the main drivers of our initial
+                project specification and help turn that specification into actionable and Agile
+                developer tickets.
+              </Typography>
+            </Box>
+          </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="h5" gutterBottom>
+                What we’re looking for
+              </Typography>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                A great project manager is someone that is very organized and has the foresight
+                to anticipate future challenges. They must be an excellent communicator and have
+                the confidence to lead and manage a team. Experience writing software is a plus,
+                but experience managing any kind of product from start to finish will be more
+                applicable. The ideal candidate is passionate about bringing projects to life
+                and is excited about empowering a team to make an impact in the Boston community.
+              </Typography>
+            </Box>
+          </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="h5" gutterBottom>
+                Our ideal candidate is someone who is...
+              </Typography>
+              <Typography variant="body1" className={classes.text}>
+                <ul className={classes.list}>
+                  <li>
+                    Organized and forward thinking
+                  </li>
+                  <li>Passionate about helping the Boston community</li>
+                  <li>
+                    A strong communicator who facilitates collaboration amongst
+                    the product team
+                  </li>
+                  <li>
+                    Interested in being a part of the entire product life cycle:
+                    software inception to completion
```

Comment:
Not sure about "software inception". Just inception could work, or maybe something like "from proposal to implementation to completion". Not a great example, but something like that? "software inception" just feels like it's not a thing

Reply:
I made it sounds nicer